### DISCUSSION TITLE
网站日志提示服务内部错误: bufio.Scanner: token too long

### DISCUSSION THREAD

**Discussion by R0A1NG** [1Panel-dev/1Panel#6097] [Q&A]
### 1Panel 版本

v1.10.13-lts

### 问题描述

日志文件26MB，访问网站日志接口报错
![1723364210307](https://github.com/user-attachments/assets/d4300b5a-ab32-490d-9d06-37328480dff9)

### 重现步骤

访问网站日志接口

**Comment by zhengkunwang223** [comment]
感谢反馈 我们先在本地环境复现一下

**Comment by zhengkunwang223** [comment]
不好意思 没有复现
![image](https://github.com/user-attachments/assets/5a4421dd-c188-479d-8a64-ed6ab081eca4)
麻烦t添加一下我的企业微信 我远程看一下

**Comment by zhengkunwang223** [comment]
已复现 后续版本修复
